resources/logging/templates/promtail/daemonset.yaml

```diff
@@ -8,71 +9,75 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
   annotations:
-    {{- toYaml .Values.promtail.annotations | nindent 4 }}
+        {{- toYaml .Values.promtail.annotations | nindent 4 }}
 spec:
   selector:
     matchLabels:
       app: {{ template "promtail.name" . }}
       release: {{ .Release.Name }}
   updateStrategy:
     type: {{ .Values.promtail.deploymentStrategy }}
-  {{- if ne .Values.promtail.deploymentStrategy "RollingUpdate" }}
+          {{- if ne .Values.promtail.deploymentStrategy "RollingUpdate" }}
     rollingUpdate: null
-  {{- end }}
+        {{- end }}
   template:
     metadata:
       labels:
         app: {{ template "promtail.name" . }}
         release: {{ .Release.Name }}
       annotations:
-        {{ toYaml .Values.promtail.podAnnotations | nindent 8 }}
+          {{ toYaml .Values.promtail.podAnnotations | nindent 8 }}
     spec:
       serviceAccountName: {{ template "promtail.serviceAccountName" . }}
-    {{- if .Values.promtail.priorityClassName }}
+            {{- if .Values.promtail.priorityClassName }}
       priorityClassName: {{ .Values.promtail.priorityClassName }}
-    {{- end }}
+            {{- end }}
       containers:
         - name: promtail
           image: "{{ .Values.promtail.image.repository }}:{{ .Values.promtail.image.tag }}"
           imagePullPolicy: {{ .Values.promtail.image.pullPolicy }}
           args:
-            - "-config.file=/etc/promtail/{{ .Values.global.logging.promtail.config.name }}"
+            {{- if (semverCompare ">=1.14.0" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
On the top you have already initialized a variable holding version why not reuse it?